### PR TITLE
Converter fixes from testing tagged BgpPeerP object.

### PR DESCRIPTION
**Pull request by shastrinator**
Added fixes from testing the tagged bgp peer prefix object which was failing.
Forward conversion should use aliased object.
Reversed conversion should pick up the same object.

### CODE REVIEW

File: aim/tests/unit/agent/aid_universes/test_converter.py [hunk 1]
```diff
@@ -3894,7 +3919,7 @@ class TestAimToAciConverterBgpAsPPeer(TestAimToAciConverterBase,
                   dn='uni/tn-t1/out-l1/lnodep-np1/lifp-ip1/'
                      'rspathL3OutAtt-[topology/pod-1/paths-101/'
                      'pathep-[eth1/1]]/peerP-[1.1.1.0/24]'),
-         _aci_obj('bgpAsP',
+         _aci_obj('bgpAsP__Peer',
                   asn=65000,
```
Comment (ivar-lazzaro) [inline aim/tests/unit/agent/aid_universes/test_converter.py]: This should be a string for consistency

File: aim/tests/unit/agent/aid_universes/test_converter.py [hunk 2]
```diff
@@ -3908,7 +3933,7 @@ class TestAimToAciConverterBgpAsPPeer(TestAimToAciConverterBase,
                   dn='uni/tn-t2/out-l1/lnodep-np1/lifp-ip1/'
                      'rspathL3OutAtt-[topology/pod-1/paths-101/'
                      'pathep-[eth1/1]]/peerP-[1.1.1.0/24]'),
-         _aci_obj('bgpAsP',
+         _aci_obj('bgpAsP__Peer',
                   asn=65001,
```
Comment (ivar-lazzaro) [inline aim/tests/unit/agent/aid_universes/test_converter.py]: Ditto here and in L3928/L3915/L3904

File: aim/tests/unit/agent/aid_universes/test_converter.py (hunk 1 above)
Comment (shastrinator) [reply aim/tests/unit/agent/aid_universes/test_converter.py]: Sure. Made these changes.

File: aim/tests/unit/agent/aid_universes/test_converter.py [hunk 3]
```diff
@@ -3836,37 +3836,62 @@ class TestAciToAimConverterBgpAsPPeer(TestAciToAimConverterBase,
...
-                              asn=0)]]
+    sample_input = [[
+        {'bgpPeerP': {
+         'attributes': {'rn': 'peerP-[1.1.1.0/24]',
+                        'dn': 'uni/tn-test_gbp/'
+                              'out-testOut1/lnodep-testNP1/lifp-testLifP1/'
+                              'rspathL3OutAtt-[topology/pod-1/paths-101/'
+                              'pathep-[eth1/1]]/peerP-[1.1.1.0/24]',
+                        'status': 'created'}, 'children': []}},
+        {'bgpAsP': {
```
Comment (kentwu1) [inline aim/tests/unit/agent/aid_universes/test_converter.py]: should we change this to 'bgpAsP__Peer' also then?

**Comment by shastrinator** [comment]
Tested the changes to be working.

File: aim/tests/unit/agent/aid_universes/test_converter.py (hunk 3 above)
Comment (kentwu1) [reply aim/tests/unit/agent/aid_universes/test_converter.py]: Pls also add a UT to cover this bgpAsP__Peer case then.

Comment (shastrinator) [reply aim/tests/unit/agent/aid_universes/test_converter.py]: Sure, sorry I missed this. I will modify this case with another commit. The second part of this UT already has the regular bgpAsP case.

**Pull request merged**